src/code_base/lognormal_GAMMA_efsa_NC.cpp

```diff
@@ -453,11 +460,11 @@ std::vector<double> lognormalGAMMA_efsa_BMD_NC::bmd_start_hybrid_extra_clean(std
 	Eigen::MatrixXd temp_mean = mean(theta2, d);
 	Eigen::MatrixXd temp_var = variance(theta2, d);
 	/////////////////////////////////////////////////////////////////////////////////////
-	double mu_zero = temp_mean(0, 0);
-	double std_zero = sqrt(temp_var(0, 0));
-	double ct_off = gsl_cdf_lognormal_Pinv(isIncreasing ? NOT_ADVERSE_P : TAIL_PROB, mu_zero, std_zero);
+	// double mu_zero = temp_mean(0, 0);
```

Comment:
compiler warnings about unusued variables. You will see similar changes across all the model files.